cdc/cdc.hh

```diff
@@ -45,4 +49,11 @@ seastar::sstring log_name(const seastar::sstring& table_name);
 
 seastar::sstring desc_name(const seastar::sstring& table_name);
 
+seastar::future<std::vector<mutation>>apply(
```

Comment:
`apply` is a very generic, non-descriptive name. What about `append_log_mutations`?
In any case, please write a doc string that explain what the function does.

Reply:
done.